docs/examples/pytorch/resnet50/main.py

```diff
@@ -80,18 +80,21 @@
 class HybridTrainPipe(Pipeline):
     def __init__(self, batch_size, num_threads, device_id, data_dir, crop, dali_cpu=False):
         super(HybridTrainPipe, self).__init__(batch_size, num_threads, device_id, seed=12 + device_id)
+        self.dali_cpu = dali_cpu
         self.input = ops.FileReader(file_root=data_dir, shard_id=args.local_rank, num_shards=args.world_size, random_shuffle=True)
         #let user decide which pipeline works him bets for RN version he runs
-        if dali_cpu:
+        if self.dali_cpu:
             dali_device = "cpu"
-            self.decode = ops.HostDecoder(device=dali_device, output_type=types.RGB)
+            # self.decode = ops.HostDecoder(device=dali_device, output_type=types.RGB)
+            self.decode = ops.HostDecoderRandomCrop(device=dali_device, output_type=types.RGB)
+            self.resize = ops.Resize(resize_shorter=crop)
```

Comment:
I think it is wrong. We should resize to (crop, crop), so
```
self.resize = ops.Resize(resize_x=crop, resize_y=crop)
```

Reply:
Right, I'll change that